specification/chaos/resource-manager/Microsoft.Chaos/preview/2024-03-22-preview/privateAccesses.json

```diff
@@ -222,6 +222,10 @@
             }
           }
         },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
```

Comment:
for put, use the azure-async-operation header instead of location. azure-async-operation header includes a payload.